### DISCUSSION TITLE
Guide Card Row Question - Doc Site

### DISCUSSION THREAD

**Discussion by bennettcc** [NCIOCPL/ncids#1372] [NCIDS Launch]
Submitted by Alex during Doc site review: 

On the Guide Card Row page (https://designsystem-dev.cancer.gov/develop/templates/guide-card-row): Should we clarify that each card doesn’t need to have the same number of links/buttons? One of the four examples on the page has differing number of links for each card, but I’m not sure if that is enough clarification or not.

**Comment by blilianyu** [comment]
We can update the overview image to show one of the Guide Cards in the 2-Guide Card Row to have less buttons like the 3-Guide Card Row. And we can update the 4 code snippets examples to show a mismatched number of buttons. 

cc: @monika-jaeger

**Reply by monika-jaeger** [reply]
Updated Overview image for team discussion [linked here](https://www.figma.com/file/8qu1psEjnehVDoZyWrH7ny/NCIDS-For-All?type=design&node-id=762%3A9327&mode=design&t=xW5MDJS4hgEDajLI-1). (scroll down to patterns)
This has one link removed from the right guide card to show the left card having 6 links and the right guide card having 4 links.
cc: @blilianyu

**Reply by blilianyu** [reply]
Good to create CR to update overview image for 2-Guide Card row Image

**Reply by blilianyu** [reply]
CR #1641 has been created

**Comment by laurelthrash** [comment]
Notified Anna and Alex on 04/24 via email of this intended change. Will pursue implementation of this solution via CR mentioned in above comments.
